functions/Copy-DbaSysDbUserObject.ps1

```diff
@@ -57,7 +57,7 @@ function Copy-DbaSysDbUserObject {
         https://dbatools.io/Copy-DbaSysDbUserObject
 
     .EXAMPLE
-        PS C:\> Copy-DbaSysDbUserObject -Source $sourceServer -Destination $destserver
+        PS C:\> Copy-DbaSysDbUserObject -Source sqlserver2014a -Destination sqlcluster
 
         Copies user objects from source to destination
```

Comment:
```suggestion
        Copies user objects found in system databases master, msdb and model from sqlserver2014a instance to the sqlcluster instance.
```

Reply:
@Splaxi please update the description of this example to be more informative...provided a suggestion if you just want to accept it.

Reply:
Thx for the suggestion!